src/test/java/examples/users/users.feature

```diff
@@ -0,0 +1,46 @@
+Feature: sample karate test script
+  for help, see: https://github.com/intuit/karate/wiki/IDE-Support
+
+  Background:
+    * url 'https://jsonplaceholder.typicode.com'
+
+  Scenario: get all users and then get the first user by id
+    Given path 'users'
+    When method get
+    Then status 200
+
+    * def first = response[0]
+
+    Given path 'users', first.id
+    When method get
+    Then status 200
+
+  Scenario: create a user and then get it by id
+    * def user =
+      """
+      {
+        "name": "Test User",
+        "username": "testuser",
+        "email": "test@user.com",
+        "address": {
+          "street": "Has No Name",
+          "suite": "Apt. 123",
+          "city": "Electri",
+          "zipcode": "54321-6789"
+        }
+      }
+      """
+
+    Given url 'https://jsonplaceholder.typicode.com/users'
+    And request user
+    When method post
+    Then status 201
+
+    * def id = response.id
+    * print 'created id is: ', id
+
+    Given path id
+    # When method get
+    # Then status 200
+    # And match response contains user
+  
```

Comment:
Why this is in your code?, this is a example from other repo
Please, deleted! :)

Reply:
Yeah, I didn't delete the example folder to check the folder structure i'd be using but it shouldn't be here in future PR to masters (but as I made new branches from development to continue with the other features its still there on #2 , #3 and #4 ).
Anyway, yes, it shouldn't go to master. Its deleted now (and I should updated the other branches / PR soon 😛)

Reply:
Great! thks